config/routes.rb

```diff
@@ -0,0 +1,11 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
+  # Can be used by load balancers and uptime monitors to verify that the app is live.
+  get 'up' => 'rails/health#show', as: :rails_health_check
+
+  # Defines the root path route ("/")
+  get '/random_greeting', to: 'greetings#random_greeting'
+  # root "posts#index"
```

Comment:
- [ ] Because everything is `inter-related` to each other and if one of them is not set properly then other things will also not work, please once you create the `root` view for your front-end application, you should add the `root` path here for your `front-end` App.js component to `render` the `random` greetings coming from the `API`. Right now we can only access the `random-greetings` from just the back-end endpoint `/random_greeting`. But your goal is to render the `greeting` on the `front-end` application.